lib/features/home/widgets/google_map.dart

```diff
@@ -209,7 +209,10 @@ class _GoogleMapViewState extends State<GoogleMapView>
             position: widget.currentLocation,
             icon: BitmapDescriptor.fromBytes(Uint8List.view(pngBytes.buffer)),
             zIndex: 1,
-            onTap: () {
+            onTap: () async {
```

Comment:
1. do not use await async in Widget
2. The issue with map marker and menu is because of closing menu in `onCameraMove`. We do not this implementation here.